cmd/provisioner-localpv/app/config.go

```diff
@@ -187,6 +215,26 @@ func (c *VolumeConfig) GetStorageType() string {
 	return stgType
 }
 
+//GetStorageType returns the BlockDeviceSelectors value configured
+// in StorageClass. Default is ""
+func (c *VolumeConfig) GetBlockDeviceSelectors() string {
+	blockDeviceSelector := c.getValue(KeyBlockDeviceSelectors)
+	if len(strings.TrimSpace(blockDeviceSelector)) == 0 {
+		return ""
+	}
+	return blockDeviceSelector
+}
+
+// GetBlockDeviceSelectorFields unmarshalls the multi-line blockDeviceSelectors string
+// value into a map of strings.
+func GetBlockDeviceSelectorFields(blockDeviceSelectors string) (map[string]string, error) {
```

Comment:
Should this be in the helper.go file as it's not bound to the volumeConfig struct?

Reply:
Yup, that will better.